db/migrate/20230803225825_container_indexes.rb

```diff
@@ -0,0 +1,25 @@
+class ContainerIndexes < ActiveRecord::Migration[6.1]
+  def change
+    # container projects page
+    add_index :container_routes, :container_project_id
+    add_index :container_services, :container_project_id
+    # didn't show much, since only had 1 record
```

Comment:
This comment doesn't help future readers, so let's remove the comment.  It's fine in the commit message and/or PR comments though)